src/app/views/login/login.component.ts

```diff
@@ -23,7 +23,7 @@ export class LoginComponent {
       .then(result => this.router.navigate(['/home']))
       .catch(errors => {
         if (errors.details) {
-          this.errorMessage = errors.details.password;
+          this.errorMessage = errors.details.password.charAt(0).toUpperCase() + errors.details.password.slice(1);
```

Comment:
I think if there is need for capitalization, it should be in the server - the one returning the text. If we need to reformat the text on the client, it should be in the css (e.g with ::first-letter)